src/managers/PlayerManager.ts

```diff
@@ -92,6 +94,18 @@ class PlayerManager {
         await api.post("/deletePlayer", requestBody);
     }
 
+    public checkMyTurn(): boolean {
+        const session: Nullable<Session> = SessionManager.get();
+        const players: Nullable<Player[]> = this.getAll();
+        const me: Nullable<Player> = this.getMe();
+        assert(session && players && me);
+        const playerCount: int = players.length;
+        if (session.turnIndex === null) {
+            return false;
+        }
+        return session.turnIndex % playerCount === me.orderIndex;
+    }
```

Comment:
Very good method, i like it... 2 minor things: 
- i would propose the name "isMyTurn", thoughts?
- Maybe move it into the sessionmanager because its mainly turn / turnIndex related?

Reply:
I thought about moving it into SessionManager too, but decided against it as it is Player specific and it really only needs turnindex from session. The rest (playerCount and orderIndex) come from playermanager....what do you think?

Reply:
Hmmm true... i think one can argue for both... the reason i would argue for the sessionmanager is because of the other methods present in each manager... playermanager (getAll, getMe, getOthers, generateName) / sessionmanager (hasStarted, join, create, start)... it fits more into the methods of the sessionmanager IMO... for me the playermanager manages only the players, tilemanager only the tiles and the sessionmanagers the rest like session logistics (join, create, start)... and we have now to decide where to put the turn logistics and for me there the closest to session/game flow

Reply:
(Technically we would need something like a game manager but till now we merged this into the session manager)

Reply:
Okay yeah you‘re right. All game things are in the SessionManager , I‘ll change it later this afternoon. Are we still going to, at some point, implement a game manager or are we sticking to session manager?

Reply:
When the scope stays ca the same i think we dont need one but if the session manager gets very big we have to look.